examples/compose/vtcompose/vtcompose.go

```diff
@@ -106,13 +138,34 @@ func main() {
 		}
 	}
 
-	for _, v := range strings.Split(*externalDbData, ";") {
+	return keyspaceInfoMap
+}
+
+func parseExternalDbData(externalDbData string) map[string]externalDbInfo {
+	externalDbInfoMap := make(map[string]externalDbInfo)
+	for _, v := range strings.Split(externalDbData, ";") {
 		tokens := strings.Split(v, ":")
 		if len(tokens) > 1 {
 			externalDbInfoMap[tokens[0]] = newExternalDbInfo(tokens[0], tokens[1], tokens[2], tokens[3], tokens[4], tokens[5])
```

Comment:
Maybe split this up into multiple lines since you've made everything nice and neat now :)

Reply:
I'm tempted to delete `newExternalDbInfo` since it _just_ initializes a struct (and the struct initialization is more readable imo, since the struct params are names), but I'm not sure what the convention is in Go.

Opted to just split this line after the `=` sign for now.